pageserver/src/tenant/timeline/walreceiver/connection_manager.rs

```diff
@@ -619,7 +716,10 @@ impl ConnectionManagerState {
         );
 
         if old_entry.is_none() {
-            info!("New SK node was added: {new_safekeeper_id}");
+            info!(
+                "New SK node was added{}: {new_safekeeper_id}",
+                if is_discovery { " (discovery)" } else { "" },
+            );
             WALRECEIVER_CANDIDATES_ADDED.inc();
```

Comment:
Please just use structured logging

```suggestion
            info!(
                ?is_discovery,
                %new_safekeeper_id,
                "New SK node was added",
            );
            WALRECEIVER_CANDIDATES_ADDED.inc();
```